neuro_evolution_ctrnn/tools/experiment.py

```diff
@@ -87,12 +87,10 @@ def _setup(self):
                                                                     input_space=self.input_space,
                                                                     output_space=self.output_space)
         logging.info("Individual size for this experiment: " + str(self.individual_size))
-        if issubclass(self.brain_class, CnnCtrnn):
-            cnn_size, ctrnn_size, cnn_output_space = self.brain_class._get_sub_individual_size(self.config.brain,
-                                                                                               input_space=self.input_space,
-                                                                                               output_space=self.output_space)
-            logging.info("cnn_size: " + str(cnn_size) + "\tctrnn_size: " + str(ctrnn_size) + "\tcnn_output: " +
-                         str(cnn_output_space))
+        info = self.brain_class.get_free_parameter_metadata(self.config.brain,
+                                                            input_space=self.input_space,
+                                                            output_space=self.output_space)
+        logging.info("consumption of free parameter: " + str(info))
```

Comment:
```suggestion
        logging.info("Composition of free parameters: " + str(info))
```

I don't like the word consumption here. I think something in the likes of Composition/Structure/Arrangement fits better here

Reply:
My thoughts for the word "Consumption" went into the direction of "Producer–consumer problem". The optimizer would be the "producer" of the data, and the brain-constructor would be the "consumer". I don't like Composition/Structure/Arrangement,  because they imply that we only re-arrange the free parameters into matrices/vectors. In practice this would be correct, but in theory we could also do other stuff with it. (In 

What would you think of "Usage of free parameters"?

Accordingly [this function](https://github.com/neuroevolution-ai/NeuroEvolution-CTRNN_new/pull/58/files#r570983463) could be called `get_free_parameter_usage`

Reply:
Usage is fine by me